### PR TITLE
Add NUnit news for 2020-07

### PR DESCRIPTION
**Pull request by jnm2**
@SeanKilleen, @OsirisTerje, @mikkelbu, @ChrisMaddock, @CharliePoole, @rprouse

Reviews appreciated. Should mention that NUnit.Analyzers 0.3.0 was released last month? Also, is it okay if I add NUnit.Analyzers to the list of projects that triggers a newsletter entry to publicize it?

### CODE REVIEW

**Comment by ChrisMaddock** [comment]
Both ideas sound good to me. 👍

**Comment by mikkelbu** [comment]
I think the current text looks good, and I don't mind you "adding NUnit.Analyzers to the list of projects that triggers a newsletter entry to publicize it" (if it means what I think it means 😄). Regarding the content of the Analyzers then I plan to do a release today or tomorrow, so please wait with the text if possible.

**Comment by mikkelbu** [comment]
@jnm2 The new version - 0.4 - has been released to nuget.

File: input/blog/posts/_current-newsletter-draft.md [hunk 1]
```diff
@@ -72,6 +72,16 @@ We are also happy to announce that [Shane Krueger](https://github.com/Shane32) a
 
 Full details are available [here](https://github.com/linq2db/linq2db/wiki/Release-Notes-3.0.0)
 
+### [NUnit](https://nunit.org)
+
+We’re grateful to have [Sean Killeen](https://seankilleen.com/) join us as lead of the documentation project. He put in a lot of effort to modernize the NUnit documentation, to great effect as you can see at [https://docs.nunit.org](https://docs.nunit.org)! The docs are integrated with the website, they now have full-text search, and each page has a link which allows you to edit on GitHub and submit a PR easily.
```
Comment (SeanKilleen) [inline input/blog/posts/_current-newsletter-draft.md:77]:
```suggestion
We’re grateful to have [Sean Killeen](https://seankilleen.com/) join us as lead of the documentation project. He put in a lot of effort to modernize the NUnit documentation, to great effect as you can see at [https://docs.nunit.org](https://docs.nunit.org)! The docs are integrated with the website, they now have full-text search, and each page has a link which allows you to edit on GitHub and submit a PR easily. We look forward to seeing [your suggestions and contributions](https://github.com/nunit/docs)! 
```

Comment (SeanKilleen) [inline input/blog/posts/_current-newsletter-draft.md:77]: Also, thanks for the shout-out. 😄

**Comment by jnm2** [comment]
@SeanKilleen @mikkelbu Updated. Any last suggestions before I flip this back from being a draft PR?

**Comment by SeanKilleen** [comment]
@jnm2 ship it! :)

**Pull request merged**